client/a8c-for-agencies/data/sites.ts

```diff
@@ -78,7 +80,9 @@ export const mockedSites: Site[] = [
 		latest_scan_threats_found: [],
 		latest_scan_has_threats_found: false,
 		jetpack_boost_scores: { desktop: 0, mobile: 0, overall: 0 },
-		php_version_num: 8.1,
+		php_version_num: '8.1',
```

Comment:
The php_version_num field is a number returned by the endpoint

Reply:
Reverted back to number in fb88090.